backend/tracim_backend/lib/core/userconfig.py

```diff
@@ -0,0 +1,27 @@
+from typing import Dict
+
+from tracim_backend.models.auth import User
+from tracim_backend.models.tracim_session import TracimSession
+from tracim_backend.models.userconfig import UserConfig
+
+
+class UserConfigApi:
+    """Api to query user configuration"""
+
+    def __init__(self, user: User, session: TracimSession):
+        self._user = user
+        self._session = session
+
+    def get_config(self) -> UserConfig:
+        query = self._session.query(UserConfig)
+        return query.filter(UserConfig.user_id == self._user.user_id).one()
+
+    def get_all_params(self) -> Dict:
```

Comment:
If you just want to say "any dict", you can use `dict` directly. `typing.Dict` is mainly useful to more precisely type the dict's contents (eg. `Dict[str, Any]` in your case).

Reply:
Indeed, thanks. I used `Dict[str, Any]`